File: ComPDFKit 1.11.0 for Windows/Examples/Samples/LicenseKey.cs
using ComPDFKit.NativeMethod;
using System;
using System.Xml;
using static ComPDFKit.NativeMethod.CPDFSDKVerifier;

public static class SDKLicenseHelper
{ 
    public static bool LicenseVerify()
    {
        if (!LoadNativeLibrary())
            return false;

        LicenseErrorCode verifyResult = CPDFSDKVerifier.LicenseVerify("license_key_windows.txt", true);
        return (verifyResult == LicenseErrorCode.E_LICENSE_SUCCESS);
    }
}

File: ComPDFKit 1.11.0 for Windows/Examples/Samples/LicenseKey.vb
Imports ComPDFKit.NativeMethod
Imports System
Imports System.Xml
Imports ComPDFKit.NativeMethod.CPDFSDKVerifier


Public NotInheritable Class SDKLicenseHelper

    Public Shared Function LicenseVerify() As Boolean
        If Not LoadNativeLibrary() Then
            Return False
        End If

        Dim verifyResult As LicenseErrorCode = CPDFSDKVerifier.LicenseVerify("license_key_windows.txt", True)
        Return (verifyResult <> LicenseErrorCode.E_LICENSE_SUCCESS)
    End Function
End Class
